core/src/main/kotlin/org/evomaster/core/problem/rest/service/AbstractRestSampler.kt

```diff
@@ -245,6 +245,38 @@ abstract class AbstractRestSampler : HttpWsSampler<RestIndividual>() {
         initAdHocInitialIndividuals()
     }
 
+    private fun getEndPointsToSkipSimilarHelper(swagger: OpenAPI)
+    :List<String>{
+
+        // all endpoints
+        val all = swagger.paths.map{it.key}
+
+        if(all.none { it == configuration.endpointFocus }){
+            throw IllegalArgumentException(
+                "Invalid endpointFocus: ${configuration.endpointFocus}. " +
+                        "\nAvailable:\n${all.joinToString("\n")}")
+        }
+
+        // filter all items which do not contain the user provided item
+        return all.filterNot { it.toString().contains(configuration.endpointFocus.toString()) }
```

Comment:
`contains` sounds bit tricky... should rather be `startsWith`? anyway, we can do some pair-programming to introduce a new variable to handle this case